game/game.go

```diff
@@ -167,6 +167,9 @@ func (g *Game) handleCommand(c playerCommand) {
 }
 
 func (g *Game) resign(p *player) {
+	if p.kibitzer {
+		return
+	}
```

Comment:
While it's required for the server to reject kibitzer recommendations, perhaps the client side should not display the resign button to kibitzers.